### ISSUE TITLE
Split DBs for operational and security reasons

### ISSUE THREAD

**Issue by riccardomurri** [TissueMAPS/TissueMAPS#88]
At the moment, TM uses a single DB for two distinct purposes:

1. Storing user login information, settings, and GC3Pie tasks (the main "tissuemaps" DB)
2. Storing experiment and object data (in the per-user "experiment_XXX") DBs)

These two require very different access and operations policies:  DB 1. is administrative data, which should only be accessed and modified by the TM administrator and does not hinge on any special DB feature (which any SQL RDMS could in principle provide); DB 2. should only be accessed by the creating user, and requires a very special set of features (sharding via Citus, spatial queries via PostGIS -- which only PostgreSQL supports).

Worse still, because of 2. the TM DB user must be granted administrator privileges to the database, in order to create and drop the additional `experiment_XXX` tables.

I think we should modify the TM server code to use (potentially) separate DBs for purposes 1. and 2.:

* a "generic" RDBMS could be used for 1., without granting any special permission to the "tissuemaps" DB user (aside from R/W)
* a "specialized" RDBMS (= PostgreSQL + Citus + PostGIS) for the experiment data; here the "tissuemaps" user would need to have admin access (or at least enough privileges to successufully run `CREATE TABLE` and `DROP TABLE`)

**Comment by hackermd** [comment]
I agree that these two parts could be separated into independent dbs. However, PostgreSQL allows management of permissions per schema. So in the short term we would not gain that much from maintaining two dbs in this respect. In the long term, it might make it easier to migrate the "experiment" part to a different distributed db though (if that makes sense).

In any case we should handle permissions better at the app as well as db level.
